src/Goerli-DssSpell.t.sol

```diff
@@ -511,530 +452,28 @@ contract DssSpellTest is GoerliDssSpellTestBase {
-
-        vow.heal(min(vat.dai(address(vow)), sub(sub(vat.sin(address(vow)), vow.Sin()), vow.Ash())));
-
-        // Removing the surplus to allow continuing the execution.
-        hevm.store(
-            address(vat),
-            keccak256(abi.encode(address(vow), uint256(5))),
-            bytes32(uint256(0))
-        );
-
-        end.thaw();
-
-        end.flow("RWA008-A");
-
-        dai.approve(address(daiJoin), 1_000_000 * WAD);
-        daiJoin.join(address(this), 1_000_000 * WAD);
-
-        vat.hope(address(end));
-        end.pack(1_000_000 * WAD);
-
-        assertEq(vat.gem("RWA008-A", address(this)), 0, "RWA008: wrong vat gem");
-        assertEq(rwagem_008.balanceOf(address(this)), 0, "RWA008: wrong gem balance");
-        end.cash("RWA008-A", 1_000_000 * WAD);
-        assertGt(vat.gem("RWA008-A", address(this)), 0, "RWA008: wrong vat gem after cash");
-        assertEq(rwagem_008.balanceOf(address(this)), 0, "RWA008: wrong gem balance after cash");
-        rwajoin_008.exit(address(this), vat.gem("RWA008-A", address(this)));
-        assertEq(vat.gem("RWA008-A", address(this)), 0, "RWA008: wrong vat gem after exit");
-        assertGt(rwagem_008.balanceOf(address(this)), 0, "RWA008: wrong gem balance after exit");
-    }
-
-    function testRWA009_INTEGRATION_BUMP() public {
-        vote(address(spell));
-        scheduleWaitAndCast(address(spell));
-        assertTrue(spell.done());
-
-        giveAuth(address(oracle), address(this));
-
-        (, address pip, , ) = oracle.ilks("RWA009-A");
-
-        assertEq(DSValueAbstract(pip).read(), bytes32(100 * MILLION * WAD), "RWA009: Bad initial PIP value");
-
-        oracle.bump("RWA009-A", 110 * MILLION * WAD);
-
-        assertEq(DSValueAbstract(pip).read(), bytes32(110 * MILLION * WAD), "RWA009: Bad PIP value after bump()");
-    }
-
-    function testRWA009_INTEGRATION_TELL() public {
-        vote(address(spell));
-        scheduleWaitAndCast(address(spell));
-        assertTrue(spell.done());
+    function testRWA009_SPELL_DRAW() public {
+        address rwaUrn009       = addr.addr("RWA009_A_URN");
+        address rwaUrn009Output = addr.addr("RWA009_A_OUTPUT_CONDUIT"); // for goerli, we use the pause proxy
 
-        giveAuth(address(vat), address(this));
-        giveAuth(address(oracle), address(this));
+        (uint256 pink, uint256 part) = vat.urns("RWA009-A", address(rwaUrn009));
+        uint256 prevBalance = dai.balanceOf(address(rwaUrn009Output));
 
-        (, , , uint48 tocPre) = oracle.ilks("RWA009-A");
-        assertEq(uint256(tocPre), 0, "RWA009: `toc` is not 0 before tell()");
-        assertTrue(oracle.good("RWA009-A"), "RWA009: Oracle not good before tell()");
+        assertEq(prevBalance, 25_000_000 * WAD, "RWA009/bad-recipient-balance-before-spell");
+        assertEq(pink, 1 * WAD,                 "RWA009/bad-art-before-spell");
+        assertEq(part, 25_000_000 * WAD,        "RWA009/bad-ink-before-spell");
```

Comment:
Similar with absolute values. Anyone can permissionlessly repay the vault debt. I would just test the difference.